src/get-started/java-binding.md

```diff
@@ -281,111 +284,86 @@ Exonum transactions allow you to perform modifying atomic operations with the
-  the service and this transaction, serialized transaction parameters
-  as a payload, and signs the message with the author’s key pair.
-2. The client transmits the message to one of the Exonum nodes in the network.
-  The transaction is identified by the hash of the corresponding message.
-3. The node verifies the correctness of the message: its header,
-  including the service ID, and its cryptographic signature
-  against the author’s public key included into it.
-4. If all checks pass, the node that received the message adds it to its
-  local transaction pool and broadcasts the message to all the other nodes
-  in the network.
-5. Other nodes, having received the transaction message, perform all
-  the previous verification steps, and, if they pass, add the message to
-  the local transaction pool.
-6. When majority of validator nodes agree to include this transaction
-  into the next block, they take the message from the transaction pool and
-  request the service to [execute](#transaction-execution) it.
-7. When all transactions in the block are executed, all changes are atomically
-  applied to the database state and a new block is committed.
-
-The transaction messages are preserved in the database regardless of
-the execution result, and can be later accessed via `Blockchain` class.
-For a more detailed description of transaction processing,
-see the [Transaction Lifecycle](../architecture/transactions.md#lifecycle)
-section.
-
-#### Transaction Execution
-
-When the framework receives a transaction message, it must transform it into
-an *executable transaction* to process. As every service has several transaction
-types each with its own parameters, it must provide
-a [`TransactionConverter`][transactionconvererter] for this purpose.
-When the framework requests a service to convert a transaction,
-its message is guaranteed to have a correct cryptographic signature.
-
-An executable transaction is an instance of a class implementing
-the [`Transaction`][transaction] interface and defining transaction
-business logic. The interface implementations must define an execution
-rule for the transaction in `execute` method.
-
-The `Transaction#execute` method describes the operations that are applied to the
-current storage state when the transaction is executed. Exonum passes
-an [execution context][transaction-execution-context] as an argument.
-The context provides:
-
-- A `Fork` – a view that allows performing modifying operations;
+- A mutable access to the blockchain data, which allows performing modifying
+  operations;
 - The service instance name and numeric ID;
 - Some information about the corresponding transaction message:
   its SHA-256 hash that uniquely identifies it, and the author’s public key.
 
 A service schema object can be used to access data collections of this service.
 
-An implementation of the `Transaction#execute` method must be a pure function,
+!!! note "Example of a simple transaction"
```

Comment:
Will we use inlined code snippets here?

Reply:
I think, eventually, yes, I'd move them out of here.